example/XmlExample.php

```diff
@@ -0,0 +1,366 @@
+<?php
+
+require_once '../vendor/autoload.php';
+
+use \FINDOLOGIC\Export\Exporter;
+use \FINDOLOGIC\Export\Data\Ordernumber;
+use \FINDOLOGIC\Export\Data\Name;
+use \FINDOLOGIC\Export\Data\Summary;
+use \FINDOLOGIC\Export\Data\Description;
+use \FINDOLOGIC\Export\Data\Price;
+use \FINDOLOGIC\Export\Data\Url;
+use \FINDOLOGIC\Export\Data\Image;
+use \FINDOLOGIC\Export\Data\Attribute;
+use \FINDOLOGIC\Export\Data\Keyword;
+use \FINDOLOGIC\Export\Data\Usergroup;
+use \FINDOLOGIC\Export\Data\Bonus;
+use \FINDOLOGIC\Export\Data\SalesFrequency;
+use \FINDOLOGIC\Export\Data\DateAdded;
+use \FINDOLOGIC\Export\Data\Sort;
+use \FINDOLOGIC\Export\Data\Property;
+
+/* Setup exporter library start */
```

Comment:
Instead of sections, why not use functions? Maybe something like:

```php
function addOrdernumbers($item)
{
    // ...
}

// Other functions...

function createExport()
{
    $exporter = Exporter::create(Exporter::TYPE_XML);

    // Maybe somehow loop the item part?
    $item = $exporter->createItem('01120c948ad41a2284ad9f0402fbc7d');

    $item = addOrdernumbers($item);
    // Call other methods to add data...

    return $exporter->serializeItems([$item], 0, 1, 1);
}
```

This way, the process of how to generate an export page is much more obvious at first glance.

Reply:
Made a xml example class